src/test/scala/query/MatchQuerySpec.scala

```diff
@@ -1,4 +1,4 @@
-package test
+package test.query
```

Comment:
why not have the tests in `com.originate.scalypher.test.*`? that way when you ship jars people depend on, you're filling up the originate specific namespace. less likely to accidentally conflict with someone else's `FooSpec`.